packages/brain/src/modules/cron/trackValidatorsPerformance/trackValidatorsPerformance.ts

```diff
@@ -8,125 +8,70 @@ import { getBlockProposalStatusMap } from "./getBlockProposalStatusMap.js";
 import { getActiveValidatorsLoadedInBrain } from "./getActiveValidatorsLoadedInBrain.js";
 import { logPrefix } from "./logPrefix.js";
 import { ConsensusClient, ExecutionClient } from "@stakingbrain/common";
+import { TotalRewards } from "../../apiClients/types.js";
 
 export async function trackValidatorsPerformanceCron({
   brainDb,
   postgresClient,
   beaconchainApi,
   executionClient,
-  consensusClient
+  consensusClient,
+  currentEpoch
```

Comment:
I would calculate here in a new function the current epoch to simplify `index.ts`. See comment form above
```ts
let lastProcessedEpoch: number | undefined = undefined; // outside the function


  try {
    const currentEpoch = await beaconchainApi.getEpochHeader({ blockId: "finalized" });
    await trackValidatorsPerformance({
      currentEpoch,
      brainDb,
      postgresClient,
      beaconchainApi,
      executionClient,
      consensusClient
    });
  } catch (e) {
    logger.error(`Error tracking validator performance: ${e}`);
  }
}
```

Reply:
done